jazz-build-module/scm-module.groovy

```diff
@@ -163,74 +164,84 @@ def setRepoPermissions(repo_owner, repo_name, admin_group) {
             returnStdout: true
         ).trim()
-    }else if(scm_config.SCM.TYPE == "bitbucket"){
-        def outputStr = sh (
-            script: "curl -X DELETE -k -u \"${scm_config.SCM.USERNAME}:${scm_config.SCM.PASSWORD}\" '" + scm_user_services_api_endpoint + repo_name +"'" ,
+    } else if (scm_config.SCM.TYPE == "bitbucket") {
+        def outputStr = sh(
+            script: "curl -X DELETE -k -u \"${scm_config.SCM.USERNAME}:${scm_config.SCM.PASSWORD}\" '" + scm_user_services_api_endpoint + repo_name + "'",
             returnStdout: true
         ).trim()
     }
 }
 
 def getRepoUrl(repo_name) {
-    if(scm_config.SCM.TYPE == "gitlab"){
+    if (scm_config.SCM.TYPE == "gitlab") {
         return "${scm_protocol}${scm_config.REPOSITORY.BASE_URL}/${scm_config.REPOSITORY.REPO_BASE_SERVICES}/${repo_name}";
-    }else if(scm_config.SCM.TYPE == "bitbucket"){
+    } else if (scm_config.SCM.TYPE == "bitbucket") {
         return "${scm_protocol}${scm_config.REPOSITORY.BASE_URL}/projects/${scm_config.REPOSITORY.REPO_BASE_SERVICES}/repos/${repo_name}/browse";
-    } 
+    }
 }
 
 def getRepoCloneUrl(repo_name) {
-    if(scm_config.SCM.TYPE == "gitlab"){
+    if (scm_config.SCM.TYPE == "gitlab") {
         return "${scm_user_services_clone_url}${repo_name}.git"
-    }else if(scm_config.SCM.TYPE == "bitbucket"){
+    } else if (scm_config.SCM.TYPE == "bitbucket") {
         return "${scm_user_services_clone_url}${repo_name}.git"
     }
 }
 
 def getRepoCloneBaseUrl(repo_name) {
-    if(scm_config.SCM.TYPE == "gitlab"){
+    if (scm_config.SCM.TYPE == "gitlab") {
         return "${scm_config.REPOSITORY.BASE_URL}/${scm_config.REPOSITORY.REPO_BASE_SERVICES}/${repo_name}.git"
-    }else if(scm_config.SCM.TYPE == "bitbucket"){
+    } else if (scm_config.SCM.TYPE == "bitbucket") {
         return "${scm_config.REPOSITORY.BASE_URL}/scm/${scm_config.REPOSITORY.REPO_BASE_SERVICES}/${repo_name}.git"
     }
 }
 
 def getRepoProtocol() {
-     return "${scm_protocol}"
+    return "${scm_protocol}"
 }
 
 def getTemplateUrl(template_name) {
-    if(scm_config.SCM.TYPE == "gitlab"){
+    if (scm_config.SCM.TYPE == "gitlab") {
         return "${scm_core_services_endpoint}${template_name}.git";
-    }else if(scm_config.SCM.TYPE == "bitbucket"){
+    } else if (scm_config.SCM.TYPE == "bitbucket") {
         return "${scm_core_services_endpoint}${template_name}.git";
     }
 }
 
 def getCoreRepoCloneUrl(repo_name) {
-    if(scm_config.SCM.TYPE == "gitlab"){
+    if (scm_config.SCM.TYPE == "gitlab") {
         return "${scm_protocol}${scm_config.REPOSITORY.BASE_URL}/${scm_config.REPOSITORY.REPO_BASE_PLATFORM}/${repo_name}.git"
-    }else if(scm_config.SCM.TYPE == "bitbucket"){
+    } else if (scm_config.SCM.TYPE == "bitbucket") {
         return "${scm_protocol}${scm_config.REPOSITORY.BASE_URL}/scm/${scm_config.REPOSITORY.REPO_BASE_PLATFORM}/${repo_name}.git"
     }
 }
 
+/**
+ * Set webhook url
+ * @return      
+ */
```

Comment:
remove this method (look at the comment above for why)

Reply:
Fixed